src/pretix/base/models/items.py

```diff
@@ -802,6 +808,12 @@ class ItemVariation(models.Model):
         verbose_name=_('Membership types'),
         blank=True,
     )
+    require_membership_hidden = models.BooleanField(
+        verbose_name=_('Hide without a valid membership'),
+        help_text=_('Do not show this unless the customer is logged in and has a valid membership. This also means this '
+                    'will never visible in e.g. the widget.'),
```

Comment:
See above.

```suggestion
        help_text=_('Do not show this unless the customer is logged in and has a valid membership. Be aware that '
                    'this means it will never be visible in the widget.'),
```